ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsIngestion.scala

```diff
@@ -88,4 +93,46 @@ private[backend] trait StorageBackendTestsIngestion
     partiesAfterLedgerEndUpdate should not be empty
   }
 
+  private val NumberOfUpsertPackagesTests = 100
```

Comment:
tests are failing locally usually after a handful of tries (if it is racy), but just to make sure... (does not take long)

this test fails for original H2 and Oracle implementation (passes for Postgres)